vendors/swedbank-pay-sdk-php/tests/InvoicePaymentTest.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+use SwedbankPay\Api\Service\Consumer\Resource\ConsumerAddress;
+use SwedbankPay\Api\Service\Invoice\Request\CreateApprovedLegalAddress;
+use SwedbankPay\Api\Service\Invoice\Request\CreateInvoice;
+use SwedbankPay\Api\Service\Invoice\Resource\Request\AddresseeObject;
+use SwedbankPay\Api\Service\Invoice\Resource\Request\Addressee;
+use SwedbankPay\Api\Service\Invoice\Resource\Request\Invoice;
+use SwedbankPay\Api\Service\Invoice\Transaction\Resource\Request\Consumer;
+use SwedbankPay\Api\Service\Invoice\Transaction\Resource\Request\Transaction;
+use SwedbankPay\Api\Service\Payment\Resource\Collection\PricesCollection;
+use SwedbankPay\Api\Service\Payment\Resource\Collection\Item\PriceItem;
+use SwedbankPay\Api\Service\Invoice\Resource\Request\PaymentPayeeInfo;
+use SwedbankPay\Api\Service\Invoice\Resource\Request\PaymentUrl;
+use SwedbankPay\Api\Service\Invoice\Resource\Request\Payment;
+use SwedbankPay\Api\Service\Invoice\Resource\Request\InvoicePaymentObject;
+
+use SwedbankPay\Api\Service\Data\ResponseInterface as ResponseServiceInterface;
+use SwedbankPay\Api\Service\Invoice\Transaction\Request\CreateAuthorization;
+use SwedbankPay\Api\Service\Invoice\Transaction\Resource\Request\Authorization;
+use SwedbankPay\Api\Service\Resource\Data\ResponseInterface as ResponseResourceInterface;
+
+class InvoicePaymentTest extends TestCase
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class InvoicePaymentTest has a coupling between objects value of 19. Consider to reduce the number of dependencies under 13.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)